.github/workflows/build_arm_whells.yml

```diff
@@ -0,0 +1,140 @@
+name: Build arm64
+
+on:
+  # for test action
+  pull_request:
+    branches:
+      - main
+      - feat/add-arm-wheel-build
+  # end for test action
+  workflow_dispatch:
+    inputs:
+      TAG_NAME:
+        description: 'Release Version Tag'
+        required: true
+  release:
+    types: [created]
+
+defaults:
+  run:
+    shell: bash -leo pipefail {0}
+
+jobs:
+  build_wheels_linux_arm64:
+    name: ${{ matrix.os }} py${{ matrix.python-version }}
+    runs-on:
+      - self-hosted
+      - ARM64
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-20.04 ]
+        # python-version: [ "3.8", "3.9", "3.10", "3.11"]
```

Comment:
if the script runs successfully, shall we enable more versions to be built?

Reply:
the script build 3.8 3.9 3.10. 3.11, look at packages/build_linux_arm64.sh line 7th.

Reply:
Should we not use the matrix values for parallel builds?

Reply:
Currently our self-hosted runner only supports one parallel job. This may be related to the fact that we only have one self-hosted runner. Also, there are some repetitive operations across different versions, such as downloading ccache and submodule caches. These operations not only consume time but also take up a considerable amount of disk space (currently one job takes up around 5GB, not to mention the intermediate files generated during the build process).

Reply:
Understood. Then perhaps we could at least use an ENV variable as list of the python builds to run, in order for this not to be 100% hardcoded?